src/gl/metal/vcMesh.mm

```diff
@@ -45,6 +42,12 @@ udResult vcMesh_Create(vcMesh **ppMesh, const vcVertexLayoutTypes *pMeshLayout,
       case vcVLT_QuadCorner:
         accumulatedOffset += 2 * sizeof(float);
         break;
+      case vcVLT_Color0:
```

Comment:
You'll need to reflect these changes below in `vcMesh_UploadData`, this file doesn't have a helper like some others.